plugins/modules/system/java_keystore.py

```diff
@@ -301,23 +356,33 @@ def __init__(self):
         self.supports_check_mode = True
         self.add_file_common_args = True
         argument_spec = dict(
-            name=dict(required=True),
-            certificate=dict(required=True, no_log=True),
-            private_key=dict(required=True, no_log=True),
-            password=dict(required=True, no_log=True),
-            dest=dict(required=True, type='path'),
-            force=dict(required=False, default=False, type='bool'),
-            private_key_passphrase=dict(required=False, no_log=True, type='str')
+            name=dict(type='str', required=True),
+            dest=dict(type='path', required=True),
+            certificate=dict(type='str', no_log=True),
+            private_key=dict(type='str', no_log=True),
+            private_key_passphrase=dict(type='str', no_log=True),
+            password=dict(type='str', required=True, no_log=True),
+            certificate_path=dict(type='path'),
```

Comment:
Can you move `certificate_path` and `private_key_path` next to the `certificate` and `private_key` options, respectively?

Reply:
done